dora/structures/models.py

```diff
@@ -197,6 +207,21 @@ class Structure(ModerationMixin, models.Model):
 
     ape = models.CharField(max_length=6, blank=True)
 
+    accesslibre_url = models.URLField(
+        verbose_name="URL accesslibre",
+        blank=True,
+        null=True,
+        validators=[validate_accesslibre_url],
+    )
+    osm_hours = models.CharField(
```

Comment:
```suggestion
    opening_hours = models.CharField(
```
ou 
```suggestion
    osm_opening_hours = models.CharField(
```
mais je ne vois pas de raison de mettre le nom du format dans le nom de la variable

Reply:
J'ai opté pour `opening_hours`